Reading_files/vascular.ipynb

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
<p>Very nice! </p><p> </p><p>However:</p><p> </p><p>1) you are defeating the purpose of reading from "../Data/" by giving the full path to your hard drive. </p><p>2) I asked you to add a time stamp to the files, so we can keep track of the version of files. Please do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>